valhalla/baldr/graphtile.h

```diff
@@ -494,7 +497,20 @@ class GraphTile {
       flow_sources = &temp_sources;
     *flow_sources = kNoFlowMask;
 
-    // TODO: current with coefficient based on distance in time from start of route
+    // TODO(danpat): this needs to consider the time - we should not use live speeds if
+    //               the request is not for "now", or we're some X % along the route
+    // TODO(danpat): for short-ish durations along the route, we should fade live
+    //               speeds into any historic/predictive/average value we'd normally use
+    if ((flow_mask & kCurrentFlowMask) && traffic_tile()) {
+      auto directed_edge_index = std::distance(const_cast<const DirectedEdge*>(directededges_), de);
+      auto live_speed = traffic_tile.getTrafficForDirectedEdge(directed_edge_index);
+      // Note: speed 0 is only valid if congestion level is high - otherwise,
+      // it could just be zeroed out memory from initialization
+      if (live_speed.speed_kmh != 0 || live_speed.congestion_level >= 4) {
```

Comment:
Could the check be `==` instead of `>=` since we haven't decided what the last few values will mean yet?